docs/kbs_attestation_protocol.md

```diff
@@ -244,50 +475,21 @@ The [JSON web token](https://jwt.io/) standard is adopted for the token. The for
 [Signature]
 ```
 
-The header part of the token declares the format standard of the token and the signature algorithm used.
-
-The payload part is divided into official claim set and custom claim set. In the official claim 
-set, we select three fields, `exp`, `iat`' and `iss`, which respectively declare the expiration 
-time, issuing time and issuer (KBS URL address) of the token. The custom field in the payload 
-contains the HW-TEE's public key sent by KBC along with the attestation evidence, which is valid 
-within the validity period of the token. When KBC uses the token to request resources or services 
-of KBS service API, KBS uses this public key recorded in the token to encrypt the symmetric key used to 
-encrypt the `output`.
-
-At the end of the token, KBS signs the header and payload with its root key to confirm that the 
-token is indeed issued by KBS itself.
-
-Within the validity period of the token, KBC can provide the token encoded by Base64 as the 
-content of the `token` field in the request to KBS, so as to quickly obtain the required 
-resources or service result.
-
-# Error information
-
-In addition to using the standard HTTPS status code to represent the returned error type, 
-it is also necessary to provide a detailed description of the attestation error. 
-We define the error return payload as follows:
-
-```json
-{
-    "error": {
-        "info": ""
-    }
-}
-```
-
-# Integration
-
-KBS uses HTTPS protocol for request and response by default. 
-The above "RCAR" attestation message can be passed as the payload of HTTPS request and response. 
-The following examples illustrate the integration of "RCAR" semantics with HTTPS request and response:
-
-For example, KBC needs to request a key, and the URL is `/kbs/key/key_id`, the steps are as follows:
+The header part of the token declares the format standard of the token and the
+signature algorithm used.
 
-1. KBC sends `GET /kbs/key/key_id` request. The request payload is the `Request` message in the "RCAR" semantics.
-2. If no token is provided in the request payload or the token is invalid, KBS returns `HTTP/1.1 200 OK` response to KBC, 
-but the payload of the response is the `Challenge` message in the "RCAR" semantics. 
-If a valid token is provided in the request payload, KBS directly jumps to step 4.
-3. KBC sends `GET /kbs/key/key_id` request to KBS again, but the payload of this request is the `Attestation` message in the "RCAR" semantics.
-4. KBS returns an `HTTP/1.1 200 OK` response to KBC and the payload is the `Response` message in the "RCAR" semantics.
+The payload part is divided into official claim set and custom claim set. In the
+official claim set, we select three fields, `exp`, `iat`' and `iss`, which
+respectively declare the expiration time, issuing time and issuer (KBS URL
+address) of the token. The custom field in the payload contains the HW-TEE's
```

Comment:
I wonder if a table would make this clearer:

| field | Description | Type | Example |
|-|-|-|-|
| `exp` | Expiration time | seconds and nanoseconds since epoch? | ? |
| `iat` | Issuing time | seconds and nanoseconds since epoch? | ? |
| `iss` | Issuer | KBS URL | ? |

Aside: Are these fields validated? And if so how? I've mentioned this before in various other places, but how are the timestamps validated, for example, to ensure they have not been spoofed? Does the overally system rely on Secure NTP for example?